app/src/main/res/values/strings.xml

```diff
@@ -106,7 +106,8 @@
     <string name="restore_restoring">Restoring backup</string>
     <string name="restore_magic_package">System package manager</string>
     <!-- This text gets shown for apps that the OS did not try to backup for whatever reason e.g. no backup was run yet -->
-    <string name="restore_app_not_eligible">Not yet backed up</string>
+    <string name="restore_app_not_yet_backed_up">Not yet backed up</string>
```

Comment:
We should revisit this string later, see if we can come up with something better. Can be a part of a string improvement PR though.